warp/yul/main.py

```diff
@@ -31,7 +32,9 @@
 AST_GENERATOR = "kudu"
 
 
-def transpile_from_solidity(sol_src_path, main_contract) -> dict:
+def transpile_from_solidity(
+    sol_src_path, main_contract, optimizers_order="VFoLRFD"
```

Comment:
Do we need to have the default value for the `optimizers_order` in two places.
If it was in one place it would be easier to keep them in sync